assets/css/custom.css

```diff
@@ -718,65 +714,84 @@ section.speakers .qcTabTitle .section-title {
 	background-size: cover;
 	min-height: 550px
 }
-
 .faq-section .toggle{
```

Comment:
Oh I thought most other sections didn't have white space between declarations. Just looked back and saw it's 50/50 some have white space, some dont. So was just trying to be consistent, but I can get rid of it?